Test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
you had pushed the file before you added it to `.gitignore ` file  Always try to add the  ignored files or folder to the `.gitignore` file before you push, so try to remove the `Test.md`  from the repo

Reply:
Yeah sure